client_example/16_convnd_fwd/CMakeLists.txt

```diff
@@ -3,3 +3,6 @@ add_executable(client_conv3d_fwd_fp32 conv3d_fwd_fp32.cpp)
 
 target_link_libraries(client_conv3d_fwd_fp16 PRIVATE composable_kernel::device_operations)
 target_link_libraries(client_conv3d_fwd_fp32 PRIVATE composable_kernel::device_operations)
+
+add_executable(client_conv3d_fwd_fp16_comp_fp8 conv3d_fwd_fp16_comp_fp8.cpp)
```

Comment:
Could you add a DTYPES check like we do in https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/develop/client_example/20_splitk_gemm/CMakeLists.txt